src/main/tests/api/ExceptionHandlerTest.java

```diff
@@ -0,0 +1,76 @@
+package api;
+
+import org.junit.Test;
+import org.mockito.internal.stubbing.BaseStubbing;
+import twitter4j.TwitterException;
+
+import javax.ws.rs.core.Response;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class ExceptionHandlerTest {
+    TwitterException mockedException = mock(TwitterException.class);
+
+    // test generic error
+    @Test
+    public void generalError(){
+        ExceptionHandler eh = new ExceptionHandler();
+        Response response = eh.ResponseBuilder(new Exception());
+        assertEquals(500, response.getStatus());
+        ErrorMessage errorMessage = (ErrorMessage) response.getEntity();
+        assertEquals(eh.GENERAL_ERROR, errorMessage.getMessage());
+    }
+    
+    // test nullpointerexception
+    @Test
+    public void nullPointerError(){
+        ExceptionHandler eh = new ExceptionHandler();
+        Response response = eh.ResponseBuilder(new NullPointerException());
+        assertEquals(500, response.getStatus());
+        ErrorMessage errorMessage = (ErrorMessage) response.getEntity();
+        assertEquals("There was an issue reading the body of your request", errorMessage.getMessage());
```

Comment:
this text is copied...what happens if the error message is updated in the class?